generator_layout.html

```diff
@@ -0,0 +1,907 @@
+                    data-print="accstmnt_approach"></ul>
+            </div>
+
+            <div data-if="accstmnt_audit_url">
+                <h3>{{ page.preview.evaluation.report }}</h3>
+                <p>
+                    {{ page.preview.evaluation.label_1 }}
+                    <span class="basic-information website-name" data-print="accstmt_namesite"
+                        data-printdefault="{{page.defaults.WEBSITE_NAME}}">{{page.defaults.WEBSITE_NAME}}</span>
+                    {{ page.preview.evaluation.label_2 }}:
+                    <a class="technical-information related-evidence" href="" data-print="accstmnt_audit_url"></a>.
+                </p>
+            </div>
+
+            <div data-if="accstmnt_audit_wcagem">
+                <h3>{{ page.preview.evaluation.report }}</h3>
+                <p>
+                    {{ page.preview.evaluation.label_3 }}
+                    <span class="basic-information website-name" data-print="accstmt_namesite"
+                        data-printdefault="{{page.defaults.WEBSITE_NAME}}">{{page.defaults.WEBSITE_NAME}}</span>
+                    {{ page.preview.evaluation.label_2 }}:
+                    <a class="technical-information related-evidence" href="" data-print="accstmnt_audit_wcagem"></a>.
+                </p>
+            </div>
+
+            <div data-if="accstmnt_audit_other">
+                <h3>{{ page.preview.evaluation.other_evidence }}</h3>
+                <p>{{ page.preview.evaluation.label_4 }} <span class="basic-information website-name" data-print="accstmt_namesite"
+                        data-printdefault="{{page.defaults.WEBSITE_NAME}}"></span> {{ page.preview.evaluation.label_2 }}:</p>
+                <ul data-print="accstmnt_audit_other" class="technical-information related-evidence-other"></ul>
+            </div>
+
+            <div id="statement-approval-block"
+                data-if="accstmnt_orginfo_approved_name, accstmnt_orginfo_approved_function">
+                <h3>{{ page.preview.approval.title }}</h3>
+
+                <p>{{ page.preview.approval.label }}:</p>
+                <p class="approval">
+                    <span class="basic-information organization-name" data-print="accstmt_org"
+                        data-printdefault="{{page.defaults.ORGANIZATION}}"></span><br />
+                    <span class="approved-by" data-print="accstmnt_orginfo_approved_name"
+                        data-printdefault="APPROVED_BY"></span><br />
+                    <span class="approved-by-jobtitle" data-print="accstmnt_orginfo_approved_function"
+                        data-if="accstmnt_orginfo_approved_function"></span>
+                </p>
+            </div>
+
+            <div data-if="accstmt_complaints">
+                <h3>{{ page.approval_and_complaints.complaints.title }}</h3>
+                <p class="complaints" data-print="accstmt_complaints"></p>
+            </div>
+
```

Comment:
Please switch order of the blocks for "approval" and "complaints" -- "complaints" should appear before "approval"
(addresses issue #109)